packages/wp-story-editor/src/components/fontCheck/index.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useEffect, useCallback, useState } from '@googleforcreators/react';
+import {
+  useStory,
+  useConfig,
+  useAPI,
+  getInUseFontsForPages,
+} from '@googleforcreators/story-editor';
+
+/**
+ * Internal dependencies
+ */
+import { useFeature } from 'flagged';
+import { FontCheckDialog } from './fontCheckDialog';
+
+export const FontCheck = () => {
+  const { dashboardLink } = useConfig();
+  const customFonts = useFeature('customFonts'); // @todo -- turn notifyDeletedFonts back on
```

Comment:
Noting --- I'm using the customFonts feature flag here so we don't need to debug "multiple" feature flags causing issues with e2e tests.  

Really customFonts are tied to this feature.